### PR TITLE
Error messages

### LINKED ISSUE #930: Proper error messages on DLL load failure
Now we could have OS- and bitness-specific messages (and if we initialize Google logging in LoadPrincipiaDll and catch exceptions there we could also look for missing C++ runtimes in a platform-specific way).

### PR DESCRIPTION
**Pull request by eggrobin**
Fix #930.

### CODE REVIEW

File: ksp_plugin_adapter/loader.cs [hunk 1]
```diff
@@ -6,40 +7,79 @@ namespace principia {
...
+  public static string LoadPrincipiaDllAndInitGoogleLogging() {
     if (loaded_principia_dll_) {
-      return true;
+      return null;
     }
     bool is_32_bit = IntPtr.Size == 4;
     String dll = null;
     Func<String, IntPtr> load;
+    Func<bool> is_cxx_installed;
+    string required_cxx_pacakges;
```
Comment (pleroy) [inline ksp_plugin_adapter/loader.cs]: packages

File: ksp_plugin_adapter/loader.cs [hunk 2]
```diff
@@ -6,40 +7,79 @@ namespace principia {
...
       case PlatformID.Unix:
         load = (name) => dlopen(name);
         if (is_32_bit) {
-          return false;
+          return "Linux 32-bit is not supported at this time.";
         } else {
           dll = @"GameData/Principia/Linux64/principia.so";
+          // TODO(egg): figure out how to check whether the right versions of
+          // libc++ and libc++abi are installed.
+          is_cxx_installed = () => false;
```
Comment (pleroy) [inline ksp_plugin_adapter/loader.cs]: Shouldn't this be `true`?

File: ksp_plugin_adapter/loader.cs [hunk 3]
```diff
@@ -6,40 +7,79 @@ namespace principia {
...
+      loaded_principia_dll_ = true;
+      Log.InitGoogleLogging();
+      return null;
+    } catch (Exception e) {
+      UnityEngine.Debug.LogException(e);
+      if (!is_cxx_installed()) {
+        return "Dependencies, namely " + required_cxx_pacakges +
+               ", were not found.";
+      } else {
+        return "An unknown error occured; detected OS " +
```
Comment (pleroy) [inline ksp_plugin_adapter/loader.cs]: occurred

File: ksp_plugin_adapter/loader.cs (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin_adapter/loader.cs]: the pacakge is a lie?

**Pull request merged**